extensions/grpc/deployment/src/test/proto/test.proto

```diff
@@ -0,0 +1,89 @@
+// Copyright 2015, Google Inc.
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are
+// met:
+//
+//     * Redistributions of source code must retain the above copyright
+// notice, this list of conditions and the following disclaimer.
+//     * Redistributions in binary form must reproduce the above
+// copyright notice, this list of conditions and the following disclaimer
+// in the documentation and/or other materials provided with the
+// distribution.
```

Comment:
We don't include it in a binary at all? (just to be extra sure)

Reply:
So no. It's only used in tests and it's part of the gRPC "kind of TCK". Zipping the sources is not considered as a binary form. We do the same on Vert.x (and gRPC Java does also the same).

However, we need to keep this copyright in the source form (which explains why it's there).